kore/src/main/scala/org/kframework/definition/outer.scala

```diff
@@ -311,6 +311,35 @@ case class Module(val name: String, val imports: Set[Module], localSentences: Se
   override def equals(that: Any) = that match {
     case m: Module => m.name == name && m.sentences == sentences
   }
+
+  def toFlatModules() : (String, Set[FlatModule]) = {
+    var flatSelf = new FlatModule(name, importedModuleNames, sentences, att)
+    var flatModules = Set(flatSelf) ++
+      (imports.size match {
+        case 0 => Set()
+        case _ => imports.map(m => m.toFlatModules._2).flatten
+      })
+    (name, flatModules)
+  }
+
+}
+
+object Import {
+  val syntaxString = "$SYNTAX"
+
+  def isSyntax(name: String): Boolean = name.endsWith("$SYNTAX")
```

Comment:
@gtrepta why are we using `"$SYNTAX"` here instead of `syntaxString` (the `val` declared two lines above)?

Reply:
Oh, that's a leftover from debugging.